### ISSUE TITLE
Optimize for Facebook

### ISSUE THREAD

**Issue by rasmuse** [demokratikollen/demokratikollen#97]
Sharing on Facebook can rather easily be improved:

https://developers.facebook.com/docs/web/share
https://developers.facebook.com/docs/plugins/checklist
https://developers.facebook.com/docs/plugins/share-button
Open Graph protocol: http://ogp.me/

**Comment by joelgoop** [comment]
And for twitter:
https://dev.twitter.com/cards/overview

**Comment by rasmuse** [comment]
Basic info for facebook and Twitter are now added. There are a couple of new variables in the jinja templates to customize images and descriptions on a per-page basis. Most pages get the same image and description, except party pages where I have created custom images. Examples:

Most pages, facebook:
![fb-general](https://cloud.githubusercontent.com/assets/1210973/6164252/94e48a52-b29d-11e4-850c-bf575a646bb4.png)

Party page, facebook:
![fb-c](https://cloud.githubusercontent.com/assets/1210973/6164255/9c407b9e-b29d-11e4-8420-fde160d7e837.png)

Twitter example:
![twitter-s](https://cloud.githubusercontent.com/assets/1210973/6164257/9c7bed0a-b29d-11e4-9f4c-35505153a481.png)
